src/main/java/io/micrometer/newrelic/NewRelicRegistry.java

```diff
@@ -260,7 +260,8 @@ private MetricBatchSender createMetricBatchSender() {
       MetricBatchSenderBuilder metricBatchSenderBuilder =
           MetricBatchSender.builder()
               .apiKey(config.apiKey())
-              .httpPoster(new MicrometerHttpPoster(httpSender));
+              .httpPoster(new MicrometerHttpPoster(httpSender))
+              .secondaryUserAgent("NewRelic-Micrometer-Exporter", implementationVersion);
```

Comment:
I think it is preferable to not append a version if it is not know. The product version is an optional part of the `User-Agent` field with [the spec stating](https://tools.ietf.org/html/rfc7231#section-5.5.3) where...

> A sender SHOULD NOT generate information in product-version that is not a version identifier

Reply:
This is here only for a local dev case...every actually published jar of this library should have a valid version in it.

Reply:
how about changing `UnknownVersion` to `dev`?

Reply:
I think we're just going to keep this as-is for now.